app/client/cli/utils_test.go

```diff
@@ -0,0 +1,66 @@
+package cli
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+)
+
+func Test_parseEd25519PrivateKeyFromReader(t *testing.T) {
```

Comment:
My personal taste leans towards table based testing because what changes is normally the Arrange part of the tests, the Act and Assert stay more or less the same and they can be generalized. If you have lots of cases it becomes very repetitive very quickly.

Of course there's the overhead of the tests "infrastructure" and the increased cognitive load but after a while it's all the same pattern. I am obviously open to exploring different ways, actually, I like the way some of the tests are written here with the repetitiveness hidden in utility functions.

I'll keep this in mind.

Addressed in https://github.com/pokt-network/pocket/pull/177/commits/ae0faa414fbddb9c8ccf1f283c3bf637634de0ee